data/mods/Brighter_Days/modinfo

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "type": "MOD_INFO",
```

Comment:
Pretty sure you committed this to the wrong branch

Reply:
> Pretty sure you committed this to the wrong branch

Yeah what's up with that? Fixing.